Bottom: 53ac6c33905b655e8e80513e7a15202383304f4f
Top:    de143d0e8db451655b9adc313a215108fad4ff34
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-04 15:01:07 +0800

Refresh of autofs-5.0.7-syncronize-handle_mounts-shutdown.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 10e78a1..955322a 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1475,7 +1475,7 @@ void finish_cond_wait(void)
 		fatal(status);
 }
 
-void handle_mounts_done(void)
+static void handle_mounts_finish(void)
 {
 	int status, cancel_state;
 
@@ -1549,7 +1549,7 @@ static void handle_mounts_cleanup(void *arg)
 	 * perform final cleanup.
 	 */
 	if (!submount)
-		handle_mounts_done();
+		handle_mounts_finish();
 
 	return;
 }
